awxkit/test/api/pages/test_base.py

```diff
@@ -0,0 +1,59 @@
+from unittest.mock import Mock, PropertyMock
+
+from http.client import NOT_FOUND
+import pytest
+
+from awxkit.api.pages import Base
+from awxkit.config import config
+
+
+@pytest.fixture(autouse=True)
+def setup_config():
+    config.credentials = {"default": {"username": "foo", "password": "bar"}}
+    config.base_url = ""
+
+
+@pytest.fixture
+def response():
+    r = Mock()
```

Comment:
It's possible to drop the `unittest` import with the following:
```suggestion
def response(mocker: MockerFixture):
    r = mocker.Mock()
```